ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/components/ReportParametersDetails.tsx

```diff
@@ -53,7 +53,7 @@ function ReportParametersDetails({ formValues }: ReportParametersDetailsProps):
     return (
         <Flex direction={{ default: 'column' }}>
             <FlexItem>
-                <Title headingLevel="h3">Report parameters</Title>
+                <Title headingLevel="h2">Report parameters</Title>
```

Comment:
Ditto `h3` versus `h2`

Reply:
Fixed. Details sections will use h2 and the wizard review step will use h3